source/_posts/2019-03-08-shipit-xiv.markdown

```diff
@@ -0,0 +1,53 @@
+It seems a bit daring to voluntarily bear the winter weather, stay up for 24-hours, complete a project start to finish, and place all other responsibilities on hold. Nevertheless, ShipIt Day always is bursting with Cerner associates prepared to take on the challenge.  *help this sentence please* Check out the video below to hear from ShipIt Day participants and supporters as to why ShipIt Day has become an engineering-wide program that supports Cerner’s development culture.
+
+<div align="center">
+  <iframe width="560" height="315" src="https://www.youtube.com/embed/uNo5666TPaA" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+</div>
+
+## The Winning ShipIt Day XIV Teams
+1st Place- Bravo Avocado (Max Schroeder, Jacob Zimmermann, Jan Monterrubio)
+
+{% img center /assets/2019-03-08-shipit-xiv/shipit-xiv-1.jpg 600px %}
+
+ >“We created "HCCgle" (pronounced "WHO-gull"), a search application for looking up HCC information by ICD-10 diagnosis code (e.g. - "E13.11") or by term (e.g. - "Diabetes mellitus"), including across physician-friendly terminologies like IMO and SNOMED CT. The app presented which HCC categories the requested term belongs to (if any) and shows how its categorization in the HCC specification has changed over time between revisions. This was created using Java, React, and DropWizard as well as an Oracle database. Our primary use case for this project is for aiding support troubleshooting, as there was not a good way to find the HCC codes used in production.” – Bravo Avocado
+
+2nd Place- Risky Salt (Kevin Eilers, Ryan Rickard, Pepper Pancoast)
+
+{% img center /assets/2019-03-08-shipit-xiv/shipit-xiv-2.jpg 600px %}
+
+>“We created an innovation/strategic roadmap prototype for Cerner's clients that shows past, current and future projects, along with crucial data points for each. The data was directly from a Microsoft SQL server that houses all of our PPM project data, and the application was built on Ruby on Rails and React.”– Risky Salt
+
+3rd Place- All the Data (Taylor Clay, Bilal Ahmad, Eric Ringle)
+
+{% img center /assets/2019-03-08-shipit-xiv/shipit-xiv-3.jpg 600px %}
+
+>“Our team worked on Prototype for a Flow Sheet- a real-time, customizable data visualization tool for patient device data. In the real world, this would be used to view trends in a patient’s health to proactively identify risks. The goal of this project was Protoype UI to support a dense flow sheet/graphical view of critical care data that would support iBus and Millennium data sources, as well as understand complexity to better gauge bringing function into solution. This was created using Node-Red (created a mock service that published data to the app) and ReactJS (the application UI).” – All the Data
```

Comment:
> Our team worked on Prototype for a Flow Sheet- a real-time, 

Our team worked on a prototype of a real-time flow sheet, that was a customizable data visualization tool for patient device data.

> The goal of this project was Protoype UI to support a dense flow sheet/graphical view of critical care data that would support iBus and Millennium data sources, as well as understand complexity to better gauge bringing function into solution.

The goal of this project was to build a prototype UI support a dense flow sheet and graphical view of critical care data. This would include Cerner [CareAware iBus](https://www.cerner.com/pages/careaware) and [Cerner Millennium](https://www.cerner.com/solutions/health-systems) data sources, as well as, understand complexity to better gauge what functions to bring into the solution. This was created using [Node-RED](https://nodered.org/) (created a mock service that published data to the app) and React (the application UI).

Reply:
Note, I don't think we need to include hyperlinks to all the technologies, especially for ones which are commonly referenced. I included Node-RED, as I see that as not as common.

Reply:
Cleaned up wording and included suggested links.